ui/apps/platform/src/utils/searchUtils.ts

```diff
@@ -237,3 +237,19 @@ export function searchValueAsArray(searchValue: ValueOf<SearchFilter>): string[]
     }
     return [searchValue];
 }
+
+/**
+ * Adds the StackRox bespoke flag for regex match, plus start-of-line and end-of-line character
+ *
+ * Non-string values will be returned unchanged.
+ * String values will return as "r/^<original>$".
+ *
+ * @param {string} item
+ * @returns {string}
+ */
+export function convertToExactMatch(item): string | unknown {
+    if (typeof item !== 'string') {
+        return item;
+    }
+    return `r/^${item}$`;
+}
```

Comment:
Curious to why we need to support non-string values. Can't this just be `export function convertToExactMatch(item: string): string {}` since the only calling function is strongly typed to use strings?

Reply:
Old habit. Some folks do defensive programming; I practice paranoid programming. Especially since TS is build-time only, it would be possible to call this function from a JS file.